backend/configs/usr/lib/cgi-bin/factory_reset.py

```
@@ -0,0 +1,51 @@
+def main():
+    form = FieldStorage(encoding="utf-8")
+    check_for_form(form)
+    check_for_confirmation(form)
+
+    # we need to update-with-reboot in order to factory reset, so we're changing output directory to .wb_update
+    RW_DIR = "/mnt/data/.wb-update/"
+    os.makedirs(RW_DIR, exist_ok=True)
+
+    # open/close it for writing to trigger wb-watch-update's special mode
+    USE_FACTORY_FIT_FLAG = os.path.join(RW_DIR, "wb_use_factory_fit.flag")
+    with open(USE_FACTORY_FIT_FLAG, "ab") as fp:
```

Ruff (0.8.2)

Local variable `fp` is assigned to but never used

Remove assignment to unused variable `fp`

(F841)